server/py/framework/utils/mlrunuvicorn.py

```diff
@@ -35,6 +35,7 @@ def run(httpdb_config, service_name="api"):
     uvicorn.run(
         f"services.{service_name}.daemon:app",
         host="0.0.0.0",
+        factory=True,
```

Comment:
why?

Reply:
this is relevant for when running service from pycharm
since .daemon:app is a function - it is implicitly mapped to be a factory and we get a warning log.
this change simply change the flag and set it explicitly.